static/js/video-apis/youtube/loadYoutubeAPI.js

```diff
@@ -0,0 +1,18 @@
+import YoutubeAPI from './YoutubeAPI';
+
+/**
+ * Loads the youtube iframe_api.
+ */
+function loadYoutubeIframeAPI() {
+  return new Promise(resolve => {
+    window.onYouTubeIframeAPIReady = () => resolve(window.YT);
```

Comment:
for my own edification, the youtube api calls this function when it loads?

Reply:
yes
https://developers.google.com/youtube/iframe_api_reference#Requirements

Reply:
unfortunately it has to live on the window